### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by OnkarRuikar**
#### Summary
As per the suggestion https://github.com/mdn/content/pull/15735#discussion_r866487094, the `domxref` macro already uses code highlighting. So, we don't have to surround the macro with backticks.

This PR fixes such redundant use of backticks.

#### Metadata
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/hmdvrdevice/setfieldofview/index.md [hunk 1]
```diff
@@ -26,9 +26,9 @@ setFieldOfView(leftFOV, rightFOV, zNear, zFar)
 ### Parameters
 
 - `leftFOV` {{optional_inline}}
-  - : A `{{domxref("VRFieldOfView")}}` object that defines the new field of view for the left eye. If not specified, the left eye field of view does not change.
+  - : A {{domxref("VRFieldOfView")}} object that defines the new field of view for the left eye. If not specified, the left eye field of view does not change.
 - `rightFOV {{optional_inline}}`
```
Comment (OnkarRuikar) [inline files/en-us/web/api/hmdvrdevice/setfieldofview/index.md:30]:
```suggestion
- `rightFOV` {{optional_inline}}
```

File: files/en-us/web/api/cssmathvalue/operator/index.md [hunk 2]
```diff
@@ -24,15 +24,15 @@ subtype is `CSSMathSum`, this property will return the string
...
-| `{{domxref('CSSMathProduct')}}` | `"product"` |
-| `{{domxref('CSSMathMin')}}`         | `"min"`     |
-| `{{domxref('CSSMathMax')}}`         | `"max"`     |
+| Interface                         | Value       |
+|-----------------------------------| ----------- |
+| {{domxref('CSSMathSum')}}         | `"sum"`     |
+| {{domxref('CSSMathProduct')}}     | `"product"` |
+| {{domxref('CSSMathMin')}}         | `"min"`     |
+| {{domxref('CSSMathMax')}}         | `"max"`     |
 | `{{domxref('CSSMathClamp')}}`     | `"clamp"`   |
```
Comment (teoli2003) [inline files/en-us/web/api/cssmathvalue/operator/index.md:33]: You forgot this one:
```suggestion
| {{domxref('CSSMathClamp')}}     | `"clamp"`   |
```

File: files/en-us/web/api/formdataentryvalue/index.md [hunk 3]
```diff
@@ -6,8 +6,8 @@ tags:
 ---
 {{APIRef("XMLHttpRequest")}}
 
-A `string` or `{{domxref("File")}}` that represents a single value from a set of `{{domxref("FormData")}}` key-value pairs.
+A `string` or {{domxref("File")}} that represents a single value from a set of {{domxref("FormData")}} key-value pairs.
 
 This type is returned by the `{{domxref("FormData.get()")}}` and `{{domxref("FormData.getAll()")}}` methods. The `{{domxref("FormData.get()")}}` method returns a single value while `{{domxref("FormData.getAll()")}}` returns an array of `FormDataEntryValue`s.
 
-Note that the `{{domxref("FormData.append()")}}` and `{{domxref("FormData.set()")}}` methods allow passing a `{{domxref("Blob")}}` value, which is converted to a `File` in the process.
+Note that the `{{domxref("FormData.append()")}}` and `{{domxref("FormData.set()")}}` methods allow passing a {{domxref("Blob")}} value, which is converted to a `File` in the process.
```
Comment (teoli2003) [inline files/en-us/web/api/formdataentryvalue/index.md:13]: Two more on this line:
```suggestion
Note that the {{domxref("FormData.append()")}} and {{domxref("FormData.set()")}} methods allow passing a {{domxref("Blob")}} value, which is converted to a `File` in the process.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Two small fixes. I would not have thought about this case: good catch!

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**